pkgs/development/haskell-modules/default.nix

```diff
@@ -6,6 +6,7 @@
 , initialPackages ? import ./initial-packages.nix
 , configurationCommon ? import ./configuration-common.nix
 , configurationNix ? import ./configuration-nix.nix
+, ...
```

Comment:
Please do not add ellipsis here. I don't want these functions to accept any random argument without an error.

Reply:
ok, I'll use `makePackgeSet` instead

Reply:
I am sorry, but that is not what I meant. Please don't accept `...` arguments *at all*. I don't want any of these functions to take random arguments without an error.